migrations/20221126222222_spdx-licenses.sql

```diff
@@ -0,0 +1,27 @@
+ALTER TABLE mods ADD COLUMN license_new varchar(2048) DEFAULT 'LicenseRef-All-Rights-Reserved' NOT NULL;
+
+UPDATE mods SET license_new = licenses.short FROM licenses WHERE mods.license = licenses.id;
+
+UPDATE mods SET license_new = 'LicenseRef-Custom' WHERE license_new = 'custom';
+UPDATE mods SET license_new = 'LicenseRef-All-Rights-Reserved' WHERE license_new = 'arr';
```

Comment:
On the frontend I think `LicenseRef` would look pretty ugly. Is there some way to map it without hardcoding?

Reply:
Easy enough to do a substring find-and-delete, imo

Reply:
got it, yeah makes sense. we can do something on the frontend for this